src/products/dto/create-review.dto.ts

```
@@ -0,0 +1,36 @@
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  IsString,
+  IsInt,
+  Min,
+  Max,
+  IsBoolean,
+  IsOptional,
```

ESLint

[error] 'IsOptional' is defined but never used.

(@typescript-eslint/no-unused-vars)